pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -176,34 +182,28 @@ func (uc *UpgradeCluster) getClusterNodeStatus(az armhelpers.AKSEngineClient, re
 					*vm.Name, uc.NameSuffix)
 				continue
 			}
-
 			currentVersion := uc.getNodeVersion(kubeClient, *vm.Name, vm.Tags)
-			if currentVersion == "" {
-				uc.Logger.Infof("Skipping VM: %s for upgrade as the orchestrator version could not be determined.", *vm.Name)
-				continue
-			}
 
-			// Skip the VM upgrade validation for managed clusters as it only applies to aks-engine version support.
-			if !uc.DataModel.Properties.IsHostedMasterProfile() {
-				if err := uc.upgradable(currentVersion); err != nil {
-					return err
+			if uc.Force {
```

Comment:
the if Force is above for VMAS, we don't need to check anything we just add it to the ToUpgrade list.

Reply:
makes sense. I didn't see the `uc.addVMToUpgradeSets` below the deleted lines...

Reply:
is that ACK @CecileRobertMichon ?